schemas/user/company_info.schema_v1.0.json

```diff
@@ -0,0 +1,164 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://ownera.io/certificates/owner/kyc.schema.json",
+  "title": "Company Owner Info",
+  "description": "Information about an owner from type company",
+  "type": "object",
+  "definitions": {
+    "country": {
+      "type": "string",
+      "enum": ["AD", "AE", "AF", "AG", "AI", "AL", "AM", "AO", "AQ", "AR", "AS", "AT", "AU", "AW", "AX", "AZ", "BA", "BB", "BD", "BE", "BF", "BG", "BH", "BI", "BJ", "BL", "BM", "BN", "BO", "BQ", "BR", "BS", "BT", "BV", "BW", "BY", "BZ", "CA", "CC", "CD", "CF", "CG", "CH", "CI", "CK", "CL", "CM", "CN", "CO", "CR", "CU", "CV", "CW", "CX", "CY", "CZ", "DE", "DJ", "DK", "DM", "DO", "DZ", "EC", "EE", "EG", "EH", "ER", "ES", "ET", "FI", "FJ", "FK", "FM", "FO", "FR", "GA", "GB", "GD", "GE", "GF", "GG", "GH", "GI", "GL", "GM", "GN", "GP", "GQ", "GR", "GS", "GT", "GU", "GW", "GY", "HK", "HM", "HN", "HR", "HT", "HU", "ID", "IE", "IL", "IM", "IN", "IO", "IQ", "IR", "IS", "IT", "JE", "JM", "JO", "JP", "KE", "KG", "KH", "KI", "KM", "KN", "KP", "KR", "KW", "KY", "KZ", "LA", "LB", "LC", "LI", "LK", "LR", "LS", "LT", "LU", "LV", "LY", "MA", "MC", "MD", "ME", "MF", "MG", "MH", "MK", "ML", "MM", "MN", "MO", "MP", "MQ", "MR", "MS", "MT", "MU", "MV", "MW", "MX", "MY", "MZ", "NA", "NC", "NE", "NF", "NG", "NI", "NL", "NO", "NP", "NR", "NU", "NZ", "OM", "PA", "PE", "PF", "PG", "PH", "PK", "PL", "PM", "PN", "PR", "PS", "PT", "PW", "PY", "QA", "RE", "RO", "RS", "RU", "RW", "SA", "SB", "SC", "SD", "SE", "SG", "SH", "SI", "SJ", "SK", "SL", "SM", "SN", "SO", "SR", "SS", "ST", "SV", "SX", "SY", "SZ", "TC", "TD", "TF", "TG", "TH", "TJ", "TK", "TL", "TM", "TN", "TO", "TR", "TT", "TV", "TW", "TZ", "UA", "UG", "UM", "US", "UY", "UZ", "VA", "VC", "VE", "VG", "VI", "VN", "VU", "WF", "WS", "YE", "YT", "ZA", "ZM", "ZW"]
+    },
+    "address": {
+      "type": "object",
+      "properties": {
+        "line1": { "type": "string", "maxLength": 33 },
+        "line2": { "type": "string", "maxLength": 33 },
+        "buildingNumber": { "type": "string", "maxLength": 15 },
+        "city": { "type": "string", "maxLength": 60 },
+        "postalCode": { "type": "string", "maxLength": 10 },
+        "country": { "$ref": "#/definitions/country" },
+        "state": { "type": "string", "maxLength": 90 }
+      },
+      "required": ["line1", "buildingNumber", "city", "postalCode", "country"]
+    },
+    "id": {
+      "type": "object",
+      "properties": {
+        "idType": { "enum": ["Passport", "Id", "SSN", "last 4 digits SSN", "Other"] },
+        "idNumber": { "type": "string", "maxLength": 20 }
+      },
+      "required": ["idType", "idNumber"]
+    },
+    "beneficialOwner": {
```

Comment:
Should beneficialOwner be of type Indivitual? are they different? Can beneficialOwner be a company?

Reply:
The structure of beneficialOwner and Individual's certificate is the same but the required fields are different